packages/hoprd/src/admin.ts

```diff
@@ -32,10 +32,12 @@ export class AdminServer {
       return true
     }
 
-    let cookies = {}
+    let cookies: ReturnType<typeof cookie.parse> | undefined = {}
```

Comment:
The idea was to not set cookies to something temporary that we will never ever use and let Typescript do the checks for us.

Reply:
```suggestion
    let cookies: ReturnType<typeof cookie.parse> | undefined
```
^ this is what I meant